src/downloadAzureProject/setupProjectFolder.ts

```diff
@@ -31,6 +31,7 @@ export async function setupProjectFolder(uri: vscode.Uri, vsCodeFilePathUri: vsc
     await setupProjectFolderParsed(resourceId, language, vsCodeFilePathUri, context, node, devContainerName);
 }
 
+//call this directly if you are staring from VSCode extension
```

Comment:
Don't think you need this comment necessarily

Reply:
Ok, I agree